ts/src/woo.ts

```diff
@@ -1076,7 +1076,7 @@ export default class woo extends Exchange {
             };
             const closeSide = (orderSide === 'BUY') ? 'SELL' : 'BUY';
             if (stopLoss !== undefined) {
-                const stopLossPrice = this.safeNumber2 (stopLoss, 'triggerPrice', 'price', stopLoss);
+                const stopLossPrice = this.safeNumber (stopLoss, 'triggerPrice', stopLoss);
```

Comment:
@ttodua we shouldn't use safeNumber here

Reply:
ahh, sorry, i was thinking i was commenting in `features` pr. let me revise your suggestion

Reply:
just saying to use safeString instead